core/Makefile

```diff
@@ -7,7 +7,11 @@ dev-install:
 	pip install -e .
 
 install:
+ifdef ARGS
+	pip install $(ARGS) .
```

Comment:
Is it necessary to check whether `ARGS` is empty? Isn't `pip install $(ARGS) .` equal to `pip install .` in that case?